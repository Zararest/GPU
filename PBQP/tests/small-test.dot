digraph Dump {
node[color=olive,fontsize=14, style=filled, shape=rectangle]
"0x61ab4987c0c0" [label = "31.8136
34.0573
0::0
"]
"0x61ab4987c170" [label = "29.4096
4.3918
0::1
"]
"0x61ab4987c300" [label = "21.5298
39.1461
2::0
"]
"0x61ab4987c3b0" [label = "4.7247
21.0252
2::1
"]
"0x61ab4987c520" [label = "8.31429
17.7748
1::0
"]
"0x61ab4987c680" [label = "41.7666
11.5932
3::0
"]
"0x61ab4987c750" [label = "45.5216
34.8974
4::0
"]
"0x61ab4987c0c0" -> "0x61ab4987c170" [label = "41.3345 10.116 
32.1017 23.1814 
"]
"0x61ab4987c300" -> "0x61ab4987c3b0" [label = "6.96518 21.3718 
19.8769 6.55476 
"]
}
