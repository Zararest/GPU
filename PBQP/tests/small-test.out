Edges: 2
2 x 2
41.3345 10.116 
32.1017 23.1814 
0x55815e3241d0 0x55815e324160
2 x 2
6.96518 21.3718 
19.8769 6.55476 
0x55815e323c20 0x55815e323cb0

Nodes: 7
0x55815e3241d0 2 x 1
31.8136
34.0573
0::0

0x55815e324160 2 x 1
29.4096
4.3918
0::1

0x55815e3240d0 2 x 1
8.31429
17.7748
1::0

0x55815e323c20 2 x 1
21.5298
39.1461
2::0

0x55815e323cb0 2 x 1
4.7247
21.0252
2::1

0x55815e324660 2 x 1
41.7666
11.5932
3::0

0x55815e324720 2 x 1
45.5216
34.8974
4::0

